[2022-05-22 03:27:33,890] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:27:33,900] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:27:33,900] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:27:33,901] {taskinstance.py:1357} INFO - Starting attempt 4 of 4
[2022-05-22 03:27:33,901] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:27:33,914] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extract_tables_file> on 2022-05-22 02:35:48.168632+00:00
[2022-05-22 03:27:33,919] {standard_task_runner.py:52} INFO - Started process 247 to run task
[2022-05-22 03:27:33,921] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'extract_tables_file', 'manual__2022-05-22T02:35:48.168632+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpkqamqjwk', '--error-file', '/tmp/tmpmef9z9gk']
[2022-05-22 03:27:33,921] {standard_task_runner.py:80} INFO - Job 66: Subtask extract_tables_file
[2022-05-22 03:27:33,964] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [running]> on host 541443bb6989
[2022-05-22 03:27:34,035] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=extract_tables_file
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T02:35:48.168632+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T02:35:48.168632+00:00
[2022-05-22 03:29:12,350] {python.py:173} INFO - Done. Returned value was: None
[2022-05-22 03:29:12,366] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=extract_tables_file, execution_date=20220522T023548, start_date=20220522T032733, end_date=20220522T032912
[2022-05-22 03:29:12,439] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 03:29:12,471] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
